wire-ios-data-model/Tests/Source/ManagedObjectContext/CoreDataMigrationActionFactoryTests.swift

```diff
@@ -22,20 +22,48 @@ import XCTest
 final class CoreDataMigrationActionFactoryTests: XCTestCase {
 
     // add version with actions here - aka custom migration
-    let excludedVersions: [CoreDataMessagingMigrationVersion] = [.version2_111, .version2_107]
+    let excludedVersions: [CoreDataMessagingMigrationVersion] = [
+        .version2_114,
+        .version2_111,
+        .version2_107
+    ]
 
-    func test_ItReturnsPreActionForVersion211() {
+    // MARK: - Version 2.114
+
+    func test_ItReturnsPreActionForVersion2114() {
+        // given
+        // when
+        let action = CoreDataMigrationActionFactory.createPreMigrationAction(for: .version2_114)
+
+        // then
+        XCTAssertNil(action)
+    }
```

Comment:
Test it does return an action and assert it doesn't?

Reply:
It asserts that we don't have any pre-action for this migration